src/main/java/org/alliancegenome/curation_api/model/entities/bulkloads/BulkLoad.java

```diff
@@ -71,7 +71,7 @@
         FORCED_RUNNING,
         
         
-
+        RUNNING,
```

Comment:
This seems like a random edit?

Reply:
Otherwise everything looks good.

Reply:
I was getting loads of errors in my terminal about RUNNING enum missing.  It's specified in the LinkML enum, so I added it - errors disappeared.

Reply:
Ahh ok... this is where a database migration needs to come in. Chnage any bulkload status or bulkloadfile status from `RUNNING` to `FINISHED`

Reply:
Hmm, removed enum again and no more errors.